### PR TITLE
Fixed always detect osx32 when in run mode. And add -n options to set…

### PR DESCRIPTION
**Pull request by bramblex**

### CODE REVIEW

File: bin/nwbuild [hunk 1]
```diff
@@ -9,7 +9,6 @@ var argv = optimist
     .usage('Usage: nwbuild [options] [path]')
 
     .alias('p', 'platforms')
-    .default('p', 'osx32,osx64,win32,win64')
```
Comment (adam-lynch) [inline bin/nwbuild]: Why not just remove `osx32` here?

File: bin/nwbuild [hunk 2]
```diff
@@ -54,9 +54,10 @@ if(!files) {
 }
 
 var options = {
+    appName: argv.name,
```
Comment (adam-lynch) [inline bin/nwbuild]: Cool, this still falls back to the name in `package.json` if it doesn't exist yeah?

File: bin/nwbuild (hunk 1 above)
Comment (bramblex) [reply bin/nwbuild]: i'am not share if someone using the old version node on osx32 system.

File: bin/nwbuild (hunk 2 above)
Comment (bramblex) [reply bin/nwbuild]: of course. https://github.com/nwjs-community/nw-builder/blob/master/lib/index.js#L185

File: bin/nwbuild (hunk 1 above)
Comment (adam-lynch) [reply bin/nwbuild]: I mean you're moving the default options somewhere else. Wouldn't it work the same if you changed this line to `.default('p', 'osx32,osx64,win32,win64')`?

Comment (zaygraveyard) [reply bin/nwbuild]: I agree with @adam-lynch, removing `osx32` here should be equivalent (unless I'm missing something).

PS: @adam-lynch you mean `.default('p', 'osx64,win32,win64')`, right?

File: bin/nwbuild [hunk 3]
```diff
@@ -74,6 +75,11 @@ if(argv.r) {
     options.currentPlatform = currentPlatform;
 }
 
+// Set default target platforms here
+// Remove osx32 from default, nwjs didn't support osx32 yet
+if (!options.platforms) {
+    options.platforms = ['osx64', 'win32', 'win63']
```
Comment (zaygraveyard) [inline bin/nwbuild]: Typo: `win63` should be `win64`

**Comment by bastimeyer** [comment]
@kessler You have merged and published the typo mentioned by @zaygraveyard

**Comment by kessler** [comment]
@bastimeyer thanks for spotting me! fixed

**Pull request merged**